clang/lib/Sema/SemaOverload.cpp

```diff
@@ -5680,8 +5680,13 @@ static ImplicitConversionSequence TryObjectArgumentInitialization(
   QualType ClassType = S.Context.getTypeDeclType(ActingContext);
   // [class.dtor]p2: A destructor can be invoked for a const, volatile or
   //                 const volatile object.
+  // Also, a static operator can be invoked for a const, volatile or const
+  // volatile object, apparently.
```

Comment:
The best wording i can find is this one https://eel.is/c++draft/over.match#funcs.general-4.sentence-5

```suggestion
  C++23 [over.match.funcs.general] p4.2 For static member functions, the implicit object parameter is considered to match any object.
```

Reply:
I checked the old version of the standard and found that this wording exists in C++98. File updated.